.github/workflows/testing.yml

```diff
@@ -19,3 +19,30 @@ jobs:
         with-submodules: true
     - name: Build
       run: docker build .
+
+  flake8:
+    name: Flake8
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Flake8
+      uses: TrueBrain/actions-flake8@master
+      with:
+        path: fetch_downloads
```

Comment:
Does the repo need a .flake8 file, to avoid compatibility issues with black? (as per https://github.com/psf/black#line-length )

Reply:
It does have a flake8? Either I am not understanding your question, or you missed that file :D

Flake8 only works in combination with black if you have a few ignores in a `.flake8`. Otherwise it will never pass :P

Reply:
So it does! But does it need a few more ignores?